env/HERA.env

```diff
@@ -42,6 +42,8 @@ elif [ $step = "waveinit" -o $step = "waveprep" -o $step = "wavepostsbs" ]; then
 
     export mpmd="--multi-prog"
     export CFP_MP="YES"
+    if [ $step = "waveprep" ]; then export MP_PULSE=0 ; fi
```

Comment:
I see you moved this line from wave JJOBS to the env files. It is a good intention, but I am not sure if this is really needed.  We are running  POE/CFP  jobs in many other steps as well and never set this environment variable.  Need to confirm with WAVE developers if it can be removed.

Reply:
@yangfanglin I saw this in the notes as a request from Steven to be moved from JJOBS to env.  MP_PULSE is set to zero in more than just the waveprep step and it was not removed from the JJOBS for waves from what I can tell.  I do not know if this is actually needed or not, I guess we just need to test without to see if things run okay without this being set?

Reply:
Leaving as is for now, can edit later after additional testing.